### ISSUE TITLE
"vercel_project_directory" data source does not parse the path properly when deploying on Windows.

### ISSUE THREAD

**Issue by Tom-Br** [vercel/terraform-provider-vercel#50]
Hi, 

I've been experimenting with Vercel deployments directly from the filepath in Terraform. However when deploying on a Windows OS the path and path_prefix are not interpretted correclty. As a result the deployment cannot find the right project structure during the build on Vercel and the code is "Not Found"

Steps to reproduce:

Terraform resources based on the guide on https://vercel.com/guides/integrating-terraform-with-vercel 

```
data "vercel_project_directory" "example_mac" {
  path = "../nextjs-terraform-demo"
}

resource "vercel_project" "example_mac" {
  name = "example-mac"
  framework = "nextjs"
}

resource "vercel_deployment" "example_mac" {
  project_id = vercel_project.example_mac.id

  files = data.vercel_project_directory.example_mac.files
  path_prefix = data.vercel_project_directory.example_mac.path
}
```

Apply the terraform on a windows machine. 

As a result the deployment has all the source files, but the files are not recognized in the Vercel Console. 

<img width="1220" alt="Screenshot 2022-07-14 at 12 58 22" src="https://user-images.githubusercontent.com/14942821/178967657-9a4bcbeb-cf24-4a42-8ad7-89299f837e38.png">

**Comment by dglsparsons** [comment]
Hey, apologies for the slow reply here. I thought I'd commented on this the other day. Thanks for reporting the issue thougH! 

It does indeed look like a pretty simple bug! I'm working on a fix at the moment and hopefully will have it shipped soon! :)

**Comment by dglsparsons** [comment]
Hey @Tom-Br, this should be fixed as of the [v0.6.2 release](https://github.com/vercel/terraform-provider-vercel/releases/tag/v0.6.2). 

Please give it a go and let me know if you encounter any more issues! :)

**Comment by Tom-Br** [comment]
@dglsparsons worked like a charm! Thanks for the quick response.
